templates/cli/docs/example.md.twig

```diff
@@ -0,0 +1,4 @@
+```sh
```

Comment:
Once we have an alias we should use it for all the examples.

Reply:
Done